common/scala/src/main/scala/org/apache/openwhisk/http/BasicRasService.scala

```diff
@@ -36,7 +37,7 @@ trait BasicRasService extends BasicHttpService {
     }
   }
 
-  val ping = path("ping") {
+  def ping: Route = path("ping") {
```

Comment:
this could stay as a `val`...

Reply:
Somehow I thought `val` cannot be overridden. Would fix that